Apps/Common/SIMCoupledSI.h

```diff
@@ -62,6 +85,29 @@ class SIMCoupledSI : public SIMCoupled<T1,T2>
 
       if ((conv = this->checkConvergence(tp,status1,status2)) <= SIM::DIVERGED)
         return false;
+
+      // Calculate aitken acceleration factor
+      if (aitken && omega0 != 0.0) {
```

Comment:
The bool `aitken` is strictly not needed. Can just use `omega0 != 0.0`to flag whether the Aitken relaxation is to be used or not.

Reply:
it is, you can do relaxation without aitken.